tests/src/api/extmark.rs

```diff
@@ -185,3 +185,28 @@ fn set_get_del_extmark() {
     let res = buf.del_extmark(ns_id, extmark_id);
     assert_eq!(Ok(()), res);
 }
+
+#[cfg(feature = "neovim-nightly")]
+#[oxi::test]
+fn virt_text_pos_inline() {
+    let mut buf = Buffer::current();
+
+    let ns_id = api::create_namespace("test");
+
+    let opts = SetExtmarkOpts::builder()
+        .virt_text([("", "")])
```

Comment:
Empty string in the second element of the tuple here causes issues for me. If I change this to anything other than `""` the test passes.

Reply:
But that shouldn't be a problem because this works

```lua
local buf = vim.api.nvim_get_current_buf()

local ns_id = vim.api.nvim_create_namespace("test")

local extmark_id = vim.api.nvim_buf_set_extmark(buf, ns_id, 0, 0,
  { virt_text = { { "", "" } }, virt_text_pos = "inline" })

local extmark = vim.api.nvim_buf_get_extmark_by_id(buf, ns_id, extmark_id, { details = true })

assert(extmark[3].virt_text_pos == "inline")
```

Reply:
Yeah, it's quite weird. It seems like it might be a serialization/deserialization-related issue and not actually nvim? Not sure though of course

Reply:
Yeah, it might. I'll try later tonight.